### DISCUSSION TITLE
SSH issues trying to log into VM machine on google cloud

### DISCUSSION THREAD

**Discussion by TypeNoneLJ** [NightscoutFoundation/xDrip#3806] [Q&A]
Hello All,

Been a while I have logged into Nightscout for various reasons but it has not been working lately. I began troubleshooting today and can not log into the VM terminal because of this error. Any help would be appreciated. 

denny

Connection via Cloud Identity-Aware Proxy Failed
Code: 4003
Reason: failed to connect to backend

Connection to VM is refused.
Please ensure that:
- VM has a firewall rule that allows TCP ingress traffic from the IP range 35.235.240.0/20, port: 22
- SSH daemon on target VM is up and running

You may be able to connect without using the Cloud Identity-Aware Proxy.

**Comment by Navid200** [comment]
Is the virtual machine running or is it stopped?
You can find that out following the first section on this page: https://navid200.github.io/xDrip/docs/Nightscout/Troubleshooting.html

**Comment by TypeNoneLJ** [comment]
It was running with the green check mark but did do a restart and it seems to have sorted itself out, thanks and sorry for the thread, should have tried that first. 

dennis

**Reply by Navid200** [reply]
Opening a discussion to ask a question is what these discussions are for.
There is no problem with that.
I am glad it is working.

**Reply by TypeNoneLJ** [reply]
BTW noticed my Nightscout was out of date. Ran into the 502 error but got everything updated and back online with the great documentation. Very grateful, thanks again for all your work!

dennis
